### ISSUE TITLE
Error when downloading subtitles

### ISSUE THREAD

**Issue by pge-01** [SickGear/SickGear#1396]
#### Branch version commit hash: d9d762b @ 0.25.50

Expectation | Result
------------ | -------------
Downloading subtitles for episodes | Error occurred when downloading subtitles.

Almost all shows are set to search for subtitles in English. This used to work reasonable well over the last few years, but results in an error since some months now: 

Error occurred when downloading subtitles -> TypeError: Object of type range is not JSON serializable.

Additional notes:
Subtitle sources set to: Addic7ed, Tvsubtitles and Thesubdb.
Opensubtitles is not configured (now) because this led to time outs / request limit issues.

[sickgear.log](https://github.com/SickGear/SickGear/files/9971970/sickgear.log)

**Comment by JackDandy** [comment]
yeah, this report makes me think the subtitles stuff is ready to be removed. 

thanks for the report.

**Comment by pge-01** [comment]
I would not be happy if it was removed all the way.
It helps that the episodes with subtitles included (in the video container) are marked, so I can see which subtitles have to be searched manually.
Removing that would result in even more work of checking all the downloads!!

**Comment by JackDandy** [comment]
Agreed.

**Comment by JackDandy** [comment]
It would only be the unmaintained Subliminal search that would be removed.
The way that existing and newly added files are used in the UI would remain.

**Comment by JackDandy** [comment]
i am getting info that the video file has something to do with this not search, i think you need to come to IRC

**Comment by pge-01** [comment]
Thanks, good to hear adding files and UI stays the same as mentioned. (No reason for panic).
IRC would be new to me, so (more then) a bit in the dark there..

**Comment by pge-01** [comment]
Hi,
I just installed an IRC client but it’s not clear how to continue.
Can we chat with GroupMe?

Greetings,
Paul
